### PR TITLE
,

### PR DESCRIPTION
**Pull request by KarolDenst**

### CODE REVIEW

File: doc/detailed_instruction.txt [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            3. type `powershell` and hit Enter.
+        b. 
+            1. use the Windows+X shortcut and choose PowerShell from the list, 
+            2. navigate to the project folder using the command `cd <path to the unzipped folder>`.
+
+    In what follows it is assumed that the current working directory was set to the project folder as indicated above.
+    The `ls` command will show 4 folders: Doc, Example, Exe, and Source.
+
+    To run the program navigate to the Exe folder by typing `cd Exe`.
+    Again using the `ls` command it can be verified that this folder contains some number of DLL files and other folders;
```
Comment (AleksyBalazinski) [inline doc/detailed_instruction.txt]: No longer true

File: doc/instruction.txt [hunk 2]
```diff
@@ -104,11 +38,6 @@ Running the application (Windows).
                 0 5 0
             This means that a clique consisting of two vertices (with indices 1 and 2) was found.
             The part of the adjacency matrix corresponding to the clique is colored red.
-        
-        Run the heuristic maximum clique finding algorithm on the C125.9 DIMACS benchmark instance with default size definition (vertex count)
```
Comment (AleksyBalazinski) [inline doc/instruction.txt:108]: Can we include it with some info that this is an additional functionality described in the report?

File: doc/detailed_instruction.txt (hunk 1 above)
Comment (KarolDenst) [reply doc/detailed_instruction.txt]: why not?

File: doc/instruction.txt (hunk 2 above)
Comment (KarolDenst) [reply doc/instruction.txt:108]: DIMACS is already described. I don't think he will even look at it

File: doc/detailed_instruction.txt (hunk 1 above)
Comment (AleksyBalazinski) [reply doc/detailed_instruction.txt]: Becouse Exe is going to be compiled with this new script and that doesn't produce any dlls
Or am I wrong here?

**Review by AleksyBalazinski — APPROVED** [review]
Impressive, very nice now let's see Paul Allens instruction

**Pull request merged**